sdk/search/Azure.Search.Documents/src/Generated/Models/AutocompleteOptions.Serialization.cs

```diff
@@ -48,17 +48,17 @@ void IUtf8JsonSerializable.Write(Utf8JsonWriter writer)
                 writer.WritePropertyName("minimumCoverage");
                 writer.WriteNumberValue(MinimumCoverage.Value);
             }
-            if (Optional.IsDefined(SearchFieldsRaw))
+            if (Optional.IsCollectionDefined(SearchFields))
             {
                 writer.WritePropertyName("searchFields");
-                writer.WriteStringValue(SearchFieldsRaw);
+                writer.(SearchFields);
             }
             writer.WritePropertyName("suggesterName");
             writer.WriteStringValue(SuggesterName);
-            if (Optional.IsDefined(Size))
+            if (Optional.IsDefined(Top))
```

Comment:
It looks the customization does not work here.

Reply:
It works, when I transfer the patch, some files are missing.